### ISSUE TITLE
Refactor all repos labels

### ISSUE THREAD

**Issue by alexcos20** [oceanprotocol/multi-repo-issue#102]
Right now, each repo has it's own set of labels and colors assigned.

We should define a complete list of labels, assign a specific color for each of those, and then update all repos.

We should always try to edit a label rather then deleting it, or else some issues are going to be left without labels

**Comment by AnaLoznianu** [comment]
Complete list of labels:

1. POA, color: #FEF2C0, description: Proof of Authority
2. Priority: Critical, color: #FF0205
3. Priority: High, color: #D93F0B
4. Priority: Low, color: #0E8A16
5. Priority: Mid, color: #FBCA04
6. Status: Blocked, color: #C5C114, description Blocked by dependency, platform requirement, etc (add comment to detail the reason)
7. Status: Devops, color: #aff9a7
8. Status: DoNotMerge, color: #e56327
9. Status: Duplicate, color: #cfd3d7, description: This issue or pull request already exists
10. Status: HelpWanted, color: #008672, description: Extra attention is needed
11. Status: InProgress, color: #0E8A16, description: Work in progress, don't merge
12. Status: Invalid, color: #e4e669, description: This doesn't seem right
13. Status: QA, color: C2E0C6
14. Status: Refined, color: #d4c5f9, description: Ready for development
15. Status: Unconfirmed, color: #e8f4a1
16. Status: WontFix, color: #ffffff, description: This will not be worked on
17. Type: Bug, color: #d73a4a, description: Something isn't working
18. Type: Dependencies, color: #0366d6, description: Pull requests that update a dependency file
19. Type: Documentation, color: #1D76DB, description: Improvements or additions to documentation
20. Type: Enhancement, color: #a2eeef, description: New feature or request
21. Type: Epic, color: #3E4B9E
22. Type: Good First Issue, color: #7057ff, description: Good for newcomers
23. Type: GraphicDesign, color: #ff7fd6
24. Type: Question, color: #d876e3, description: Further information is requested
25. Tool: Blockscout, color: #c6594f
26. Tool: Greenkeeper, color: #00c775
27. Tool: Kubernetes, color: #ccceff

**Comment by mihaisc** [comment]
When would these be used?

> Status: QA, color: C2E0C6
Status: Refined, color: #d4c5f9, description: Ready for development

**Comment by AnaLoznianu** [comment]
Refined is a tag that we can add to an issue once we have groomed it and is ready for development. 
QA is a tag mostly used for issues that need testing

**Comment by mihaisc** [comment]
So we start working on an issue only after it has the refined tag on it?

**Comment by AnaLoznianu** [comment]
On certain types of issues, yes, I think this should be the flow. For stories that are self-explanatory or bugs, we don't need to wait to have the refined tag before start working on it.
